operations/README.md

```diff
@@ -139,7 +139,7 @@ Additional links to power off sub-procedures provided for reference. Refer to th
 - [Prepare the System for Power Off](power_management/Prepare_the_System_for_Power_Off.md)
 - [Shut Down and Power Off Compute and User Access Nodes](power_management/Shut_Down_and_Power_Off_Compute_and_User_Access_Nodes.md)
 - [Save Management Network Switch Configuration Settings](power_management/Save_Management_Network_Switch_Configurations.md)
-- [Power Off Compute and IO Cabinets](power_management/Power_Off_Compute_and_IO_Cabinets.md)
+- [Power Off Compute Cabinets](power_management/Power_Off_Compute_Cabinets.md)
```

Comment:
This link does not work. Need to move the old file, Power_Off_Compute_and_IO_Cabinets.md
to its new name: Power_Off_Compute_Cabinets.md and edit the title in the moved file.